src/glmtools.jl

```diff
@@ -39,7 +39,7 @@ mueta(::LogLink, η) = exp(η)
 
 linkfun(::ProbitLink, μ) = -sqrt2 * erfcinv(2μ)
 linkinv(::ProbitLink, η) = erfc(-η / sqrt2) / 2
-mueta(::ProbitLink, η) = exp(-η^2 / 2) / sqrt2π
+mueta(::ProbitLink, η) = exp(-abs2(η) / 2) / sqrt2π
```

Comment:
I find it easier to read this way.  `-η^2` has too many operators to parse for me.